### PR TITLE
Movies

### PR DESCRIPTION
**Pull request by VladimirUman**

### CODE REVIEW

File: src/controllers/movies.js [hunk 1]
```diff
@@ -1,138 +1,103 @@
...
-
-        return router;
-    }
+    static async createMovie(req, res) {
+        const movie = new Movie(req.body);
 
-    createMovie(req, res) {
-        const body = req.body;
+        try {
+            await MovieService.create(movie);
```
Comment (MaxLogvyniukFive) [inline src/controllers/movies.js:9]: Does this method not return anithing?

File: src/controllers/movies.js [hunk 2]
```diff
@@ -1,138 +1,103 @@
...
 
-        if (!body) {
-            return res.status(400).json({
-                success: false,
-                error: 'You must provide a movie'
+            return res.status(201).json({
+                success: true,
```
Comment (MaxLogvyniukFive) [inline src/controllers/movies.js:12]: Move object in variable

File: src/controllers/movies.js [hunk 3]
```diff
@@ -1,138 +1,103 @@
...
-                })
-                .catch((error) => {
-                    return res.status(404).json({
-                        error,
-                        message: 'Movie not updated!'
-                    });
-                });
-        });
+
+            await MovieService.update(movie);
```
Comment (MaxLogvyniukFive) [inline src/controllers/movies.js:38]: This method also not return anything?

File: src/controllers/movies.js [hunk 4]
```diff
@@ -1,138 +1,103 @@
...
-    }
+            await MovieService.delete(movie._id);
 
-    async getMovieById(req, res) {
-        await Movie.findOne({ _id: req.params.id }, (err, movie) => {
-            if (err) {
-                return res.status(400).json({ success: false, error: err });
-            }
+            return res.status(200).json({
+                success: true,
```
Comment (MaxLogvyniukFive) [inline src/controllers/movies.js:63]: Variable for object

**Review by MaxLogvyniukFive — CHANGES_REQUESTED** [review]

File: src/controllers/movies.js (hunk 1 above)
Comment (VladimirUman) [reply src/controllers/movies.js]: fixed

File: src/controllers/movies.js (hunk 2 above)
Comment (VladimirUman) [reply src/controllers/movies.js:12]: fixed

File: src/controllers/movies.js (hunk 3 above)
Comment (VladimirUman) [reply src/controllers/movies.js]: fixed

File: src/controllers/movies.js (hunk 4 above)
Comment (VladimirUman) [reply src/controllers/movies.js:67]: fixed

**Review by MaxLogvyniukFive — APPROVED** [review]

**Pull request merged**